models/model.js

```diff
@@ -47,7 +49,31 @@ exports.fetchArticleComments = (article_id) => {
     return db.query("SELECT * FROM comments WHERE article_id = $1", [article_id])
     .then((result) => {
         if (result.rows[0] === undefined){
-            return Promise.reject({status: 404, msg: "Article not found"})
+            return Promise.reject({status: 404, msg: "There are no comments for the requested article"})
         } else return result.rows;
     })
-}
\ No newline at end of file
+}
+
+exports.insertComment = (article_id, comment) => {
+    const { body, username } = comment
+    if (/[^\d]/gi.test(article_id)) {
+        return Promise.reject(({status: 400, msg: "Invalid article_id"}))
```

Comment:
SQL will throw an error if passed the wrong data type so we dont need to be testing this ourselves as this is repeated functionality.